### PR TITLE
fix: not working when paths in config contains spaces

### PR DESCRIPTION
**Pull request by L2jLiga**

### CODE REVIEW

File: lib/forwarder.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export async function forwardToDaemon(resolver, config) {
     JSON.stringify(process.cwd()),
     JSON.stringify(process.argv)
   ];
-  socket.write(args.join(' '));
+  socket.write(JSON.stringify(args));
```
[the remark this answers is not in the document]
Comment (L2jLiga) [reply lib/forwarder.js:25]: I think yes

[the remark this answers is not in the document]
Comment (L2jLiga) [reply lib/forwarder.js:25]: fixed

**Comment by L2jLiga** [comment]
> Thank you! Can you please look into the failing tests?

Sure, fixed, tests are green on local machine

**Comment by mantoni** [comment]
Nice! Thank you ❤️

**Comment by mantoni** [comment]
📦 Released in `v14.0.2`

**Pull request merged**